synapse/glob.py

```diff
@@ -1,62 +1,85 @@
-from __future__ import annotations
-
+import asyncio
 import threading
 
-lock = threading.RLock()
+_glob_loop = None
+_glob_thrd = None
 
-plex = None     # type: 'synapse.lib.plex.Plex'
-pool = None     # s_threads.Pool(maxsize=tmax)
+def initloop():
```

Comment:
i do want to double check the behavior of making the loop this way + pytest-asyncio interaction, since we've discussed moving tests to using that for test running.